template.py

```diff
@@ -108,13 +108,13 @@
     "3.9": {
         "reasons": """
                     <li><a href="https://github.com/jugmac00/python-version-cheat-sheet#python-310">use <code>match</code> statement and write union types as <code>X | Y</code></a></li>
-                    <li><a href="https://docs.python.org/3/whatsnew/3.9.html">And more!</a></li>
+                    <li><a href="https://docs.python.org/3/whatsnew/3.10.html">And more!</a></li>
```

Comment:
Good find!

Rather than just fixing, as I'll probably make the same mistake in the future, can we futureproof this via templating?

For example for all of these:

```suggestion
                    <li><a href="https://docs.python.org/3/whatsnew/{{next_version}}.html">And more!</a></li>
```
And then check it's replaced somewhere below.

Reply:
> Rather than just fixing, as I'll probably make the same mistake in the future, can we futureproof this via templating?

Indeed that would be better but I only have time (at least for now) to contribute this very small fix rather than the proposed future proof templating option.

Reply:
Okay, thanks for the fix!

Reply:
(Templated in https://github.com/hugovk/drop-python/commit/7de7b99bcd2ae2d965e3c9847ffad22dc7ed91fa)